superset-frontend/src/features/home/Menu.tsx

```diff
@@ -306,11 +306,15 @@ export function Menu({
             arrowPointAtCenter
           >
             {isFrontendRoute(window.location.pathname) ? (
-              <GenericLink className="navbar-brand" to={brand.path}>
+              <GenericLink
+                className="navbar-brand"
+                to={brand.path}
+                tabIndex={-1}
+              >
                 <img src={brand.icon} alt={brand.alt} />
               </GenericLink>
             ) : (
-              <a className="navbar-brand" href={brand.path}>
+              <a className="navbar-brand" href={brand.path} tabIndex={-1}>
```

Comment:
I'm removing these tabs because they were out of scope. They currently will work and open the homepage when someone tabs and hits enter, but the logo gets larger and distorted when focused and doesn't highlight very well, and the functionality (sending someone to the homepage) for someone who will be using keyboard navigation on this page is not really helpful. We can add it in later, or in this PR if someone feels it's an important step.